crates/wasm/README.md

```diff
@@ -4,7 +4,6 @@
 
 In order to build the wasm package, the following must be installed:
 
-- [wasm-pack](https://github.com/rustwasm/wasm-pack)
 - [jq](https://github.com/stedolan/jq)
```

Comment:
```suggestion
```
This shouldn't be needed anymore once we are updating the versions via release-please